ui/src/installer/index.css

```
@@ -7,20 +7,20 @@ body {
     margin: 0;
 }
 
-body,
-button {
+.installer body,
```

Well, at least this one does not make sense, as you would have to put the `installer` class on `html` tag in order for this to work... I would propose to have it as part of the `installer` class then.
```suggestion
.installer,
```